### ISSUE TITLE
unable to access portfolio data on Rbbg

### ISSUE THREAD

**Issue by christianWilson** [Rblp/Rblpapi#199]
Error message NOT_APPLICABLE_TO_PORT_DATA appears whenever I try and access portfolio data through bds

**Comment by wmorgan85** [comment]
You should use getPortfolio not bds. In Excel the Bloomberg implementation
of bds routes these fields to a different service. In Rblpapi we introduced
a new function.

**Comment by eddelbuettel** [comment]
The request here was mostly invalid and a) incomplete and hence b) not reproducible.   Please do better.

**Comment by christianWilson** [comment]
Thank you for your messages.

GetPortfolio doesn't appear as a command when I run Rblpapi either

"Error in getPortfolio("TS-PX1419-24 Client", "PORTFOLIO_MEMBERS") : 
  could not find function "getPortfolio_Impl""

**Comment by christianWilson** [comment]
is I type Rblpapi::

 a list of possible commands appear: getBars, getTicks etc
but getPortfolio does not

**Comment by eddelbuettel** [comment]
Looks like this was added after the 0.3.4 release so you would have to build from source.

**Comment by wmorgan85** [comment]
One other point to note is that this is an AIM portfolio, as indicated by the TS- prefix of the security. IIRC it is not supported using the same service. Instead it is a separate product called TS-ADF I believe...

**Comment by test4GitHub** [comment]
You need to use Alpha instead of Client for an AIM portfolio. For example:
getPortfolio("TS-PX1419-24 Alpha", "PORTFOLIO_MEMBERS")

**Comment by fonz88** [comment]
I'm interested in pullying out portfolio data for a specific date. I'm using the excel function 

=+BDS("<PortfolioID> Client";"PORTFOLIO_DATA"; ;"Reference date="20170830")

But I can't implement something similar in R.

I'm using overrd = c("START_DT" = "20170828")
and calling the method port = getPortfolio(security = portID,field = DataField,overrides = ovrrd)

However, I'm retrieving the portfolio holdings for Today.

Any way to specify the date for portfolio holdings to be retrieved?

Thanks!

**Comment by eddelbuettel** [comment]
Ask Bloomberg Help to show you it is done via the (documented, supported) C++ API.  

With a working C++ solution, come here and we integrate.  Until then, please don't mistake a few colleague volunteering their time in an open source project for the Bloomberg Help desk.

**Comment by joel23888** [comment]
@fonz88 see my comment in #238 where it describes how to override the reference date.

**Comment by eddelbuettel** [comment]
And I should note that @joel23888 is a saint.

**Comment by fonz88** [comment]
@joel23888  Thanks for the reply.
 @eddelbuettel I'm afraid is not my intention to use this platform as a Blomberg help Desk.
As a new user of Rblpapi, I'm still trying to find complete documentation on the full list of function parameters.

Best Regards

**Comment by racoon971** [comment]
Hi Wehere in bloomberg termnail do you get the  TS- prefix  of an AIM portfolio ?

**Comment by armstrtw** [comment]
You need the bbg helpdesk for that question.  We are users just like you.

**Comment by racoon971** [comment]
Yes it did this , the answer is : the account number can be found under AIM>>Accounts under the num column the full TS code will be TSPXZZZZ-HHH as an example with pricine number ZZZZZ and account number HHH
